### ISSUE TITLE
bug: `java.lang.IllegalArgumentException: null` at `at net.consensys.linea.zktracer.module.hub.fragment.account.AccountFragment.<init>`

### ISSUE THREAD

**Issue by fab-10** [Consensys/besu-sequencer-plugins#1053]
Current version of the tracer is breaking some sequencer ATs, specifically this test `ProfitableTransactionTest` that when the tx is processed for block inclusion this exception is thrown.

Step to reproduce: 
- checkout the `linea-tracer` branch `fix/issue-1053/reproduce`
- run `./gradlew clean jar publishToMavenLocal` to build and install locally the tracer artifact, so it can be used in the sequencer
- checkout the `linea-sequencer` branch `reproduce-tracer-issue-1053`
- run `./gradlew clean jar`
- run `./gradlew acceptanceTests --tests linea.plugin.acc.test.ProfitableTransactionTest` it should fail reporting the error below in the report

```
17:43:02.661 [MinerExecutor] ERROR net.consensys.linea.zktracer.ZkTracer -- First exception that was caught while tracing:
java.lang.IllegalArgumentException: null
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:129)
	at net.consensys.linea.zktracer.module.hub.fragment.account.AccountFragment.<init>(AccountFragment.java:99)
	at net.consensys.linea.zktracer.module.hub.fragment.account.AccountFragment$AccountFragmentFactory.make(AccountFragment.java:79)
	at net.consensys.linea.zktracer.module.hub.section.TxSkippedSection.resolvePostTransaction(TxSkippedSection.java:130)
	at net.consensys.linea.zktracer.module.hub.defer.DeferRegistry.resolvePostTransaction(DeferRegistry.java:126)
	at net.consensys.linea.zktracer.module.hub.Hub.traceEndTransaction(Hub.java:531)
	at net.consensys.linea.zktracer.ZkTracer.traceEndTransaction(ZkTracer.java:239)
	at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:533)
	at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:265)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.processTransaction(BlockTransactionSelector.java:335)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.evaluateTransaction(BlockTransactionSelector.java:247)
	at org.hyperledger.besu.ethereum.eth.transactions.layered.LayeredPendingTransactions.selectTransactions(LayeredPendingTransactions.java:340)
	at org.hyperledger.besu.ethereum.eth.transactions.TransactionPool.selectTransactions(TransactionPool.java:562)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.lambda$timeLimitedSelection$0(BlockTransactionSelector.java:186)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
17:43:02.659 [MinerExecutor] WARN org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector -- Error during block transaction selection
java.util.concurrent.ExecutionException: 
java.lang.IllegalArgumentException
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:129)
	at net.consensys.linea.zktracer.module.hub.fragment.account.AccountFragment.<init>(AccountFragment.java:99)
	at net.consensys.linea.zktracer.module.hub.fragment.account.AccountFragment$AccountFragmentFactory.make(AccountFragment.java:79)
	at net.consensys.linea.zktracer.module.hub.section.TxSkippedSection.resolvePostTransaction(TxSkippedSection.java:130)
	at net.consensys.linea.zktracer.module.hub.defer.DeferRegistry.resolvePostTransaction(DeferRegistry.java:126)
	at net.consensys.linea.zktracer.module.hub.Hub.traceEndTransaction(Hub.java:531)
	at net.consensys.linea.zktracer.ZkTracer.traceEndTransaction(ZkTracer.java:239)
	at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:533)
	at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:265)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.processTransaction(BlockTransactionSelector.java:335)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.evaluateTransaction(BlockTransactionSelector.java:247)
	at org.hyperledger.besu.ethereum.eth.transactions.layered.LayeredPendingTransactions.selectTransactions(LayeredPendingTransactions.java:340)
	at org.hyperledger.besu.ethereum.eth.transactions.TransactionPool.selectTransactions(TransactionPool.java:562)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.lambda$timeLimitedSelection$0(BlockTransactionSelector.java:186)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.timeLimitedSelection(BlockTransactionSelector.java:189)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.buildTransactionListForBlock(BlockTransactionSelector.java:172)
	at org.hyperledger.besu.ethereum.blockcreation.AbstractBlockCreator.selectTransactions(AbstractBlockCreator.java:386)
	at org.hyperledger.besu.ethereum.blockcreation.AbstractBlockCreator.createBlock(AbstractBlockCreator.java:220)
	at org.hyperledger.besu.ethereum.blockcreation.AbstractBlockCreator.createBlock(AbstractBlockCreator.java:148)
	at org.hyperledger.besu.ethereum.blockcreation.AbstractBlockCreator.createBlock(AbstractBlockCreator.java:134)
	at org.hyperledger.besu.ethereum.blockcreation.BlockMiner.mineBlock(BlockMiner.java:143)
	at org.hyperledger.besu.consensus.clique.blockcreation.CliqueBlockMiner.mineBlock(CliqueBlockMiner.java:73)
	at org.hyperledger.besu.ethereum.blockcreation.BlockMiner.run(BlockMiner.java:83)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: net.consensys.linea.zktracer.ZkTracer$TracingExceptions: Exceptions triggered while tracing:
  - null

	at net.consensys.linea.zktracer.ZkTracer.maybeThrowTracingExceptions(ZkTracer.java:327)
	at net.consensys.linea.zktracer.ZkTracer.getModulesLineCount(ZkTracer.java:332)
	at net.consensys.linea.sequencer.txselection.selectors.TraceLineLimitTransactionSelector.evaluateTransactionPostProcessing(TraceLineLimitTransactionSelector.java:140)
	at net.consensys.linea.sequencer.txselection.selectors.LineaTransactionSelector.evaluateTransactionPostProcessing(LineaTransactionSelector.java:117)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.evaluatePostProcessing(BlockTransactionSelector.java:320)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.evaluateTransaction(BlockTransactionSelector.java:249)
	at org.hyperledger.besu.ethereum.eth.transactions.layered.LayeredPendingTransactions.selectTransactions(LayeredPendingTransactions.java:340)
	at org.hyperledger.besu.ethereum.eth.transactions.TransactionPool.selectTransactions(TransactionPool.java:562)
	at org.hyperledger.besu.ethereum.blockcreation.txselection.BlockTransactionSelector.lambda$timeLimitedSelection$0(BlockTransactionSelector.java:186)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	... 3 common frames omitted
```

**Comment by OlivierBBB** [comment]
Hi @fab-10 , I would be interested if this still happens with the current version of the tracer (we have fixed many of the `Preconditions.CheckArgument(...)` related errors.

**Comment by fab-10** [comment]
just verified that this happens also using the current tip of the `arith-dev` branch, commit https://github.com/Consensys/linea-tracer/commit/1884ceeb52e754634ac2e4bfa146f583287fc174

**Comment by fab-10** [comment]
still present on release `0.5.2-beta`

**Comment by letypequividelespoubelles** [comment]
fixed in 0.5.3-beta
